mosquitto/plan.sh

```diff
@@ -0,0 +1,38 @@
+pkg_name=mosquitto
+pkg_origin=core
+pkg_version="1.4.14"
+pkg_upstream_url="https://mosquitto.org"
+pkg_description="An Open Source MQTT v3.1/v3.1.1 Broker"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('EPL-1.0' 'EDL-1.0')
```

Comment:
I don't see EDL-1.0 isn't listed in https://spdx.org/licenses/  I'm looking in to what the correct way to handle this license is.

Reply:
Yeah, I didn't exactly know what to do with this one...

Reply:
Could you change the `EDL-1.0` to `Eclipse Distribution License - v 1.0`? 

https://www.habitat.sh/docs/reference/   under pkg_license list that as the correct way to handle licenses that are custom, or not listed in SPDX in this case.

Reply:
Done, thanks.